### PR TITLE
Contribute: Run a ZAP Hackathon

### PR DESCRIPTION
**Pull request by psiinon**
This was suggested at AppSec USA 😁 

Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

File: site/content/docs/contribute/run-a-zap-hackathon.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+title: "Contribute: Run a ZAP Hackathon"
+type: page
+---
+
+If you run a technical group like an [OWASP Local Chapter](https://owasp.org/chapters/) then how about running a ZAP
+[Hackathon](https://en.wikipedia.org/wiki/Hackathon)?
+
+You can do it all on you own but if you reach out to the [ZAP Team](/docs/team/) (either directly or via
```
Comment (kingthorin) [inline site/content/docs/contribute/run-a-zap-hackathon.md:9]: your own 
(With an r)

File: site/content/docs/contribute/run-a-zap-hackathon.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+the [User Group](https://groups.google.com/group/zaproxy-users)) then we will be delighted to help you
+as much as we can.
+
+### Setup
+
+Participants who want to change the ZAP code will need to set up a development environment - for that see the
+[ZAP Developer Guide](/docs/developer/).
+You might like to suggest that those people set up this up prior to the event.
```
Comment (kingthorin) [inline site/content/docs/contribute/run-a-zap-hackathon.md:17]:
```suggestion
You might like to suggest that those people set this up prior to the event.
```

**Review by kingthorin — COMMENTED** [review]
Great idea

**Comment by psiinon** [comment]
Thanks - I always seem to leave the "r" off "your" and of course the spelling checker doesnt flag it 😛

File: site/content/docs/contribute/run-a-zap-hackathon.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+### Easy Non-Coding Tasks
+
+Both the [documentation](../docs/) and [translations](../translate/) are great options for people who do not want to work on 
+technical tasks.
+
+### Scan Rules
+
+If you have people with more security experience then helping to improve the [scan rules](../scan-rules) 
+could be an ideal task, and one that could be very collaborative,
```
Comment (thc202) [inline site/content/docs/contribute/run-a-zap-hackathon.md:35]:
```suggestion
could be an ideal task, and one that could be very collaborative.
```

File: site/content/docs/contribute/run-a-zap-hackathon.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+If you have some pentesters who are not so interested in coding then they could [test](../testing) ZAP against one of the many 
+[deliberately vulnerable web applications](https://owasp.org/www-project-vulnerable-web-applications-directory/)). 
+
+They could then report any False Positives and False Negative they find to any people interest in coding, who
```
Comment (thc202) [inline site/content/docs/contribute/run-a-zap-hackathon.md:40]:
```suggestion
They could then report any False Positives and False Negatives they find to any people interested in coding, who
```

File: site/content/docs/contribute/run-a-zap-hackathon.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+could then look at improving the existing [scan rules](../scan-rules) or creating new ones.
+
+If you focus on a specific app which we do not currently [regularly test ZAP against](/docs/scans/) then
+you could even work out how to automated ZAP to test and score this app and submit a PR to include it in that set!
```
Comment (thc202) [inline site/content/docs/contribute/run-a-zap-hackathon.md:44]:
```suggestion
you could even work out how to automate ZAP to test and score this app and submit a PR to include it in that set!
```

**Comment by psiinon** [comment]
Thanks!
Makes it look like this PR was written while still jetlagged sitting on a hotel bed.
Oh wait...

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by kingthorin — APPROVED** [review]

**Pull request merged**